static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-love-boat.js

```diff
@@ -0,0 +1,143 @@
+define([
+    'bean',
+    'common/modules/commercial/contributions-utilities',
+    'common/utils/template',
+    'common/utils/config',
+    'common/utils/$',
+    'common/views/svgs',
```

Comment:
We're trying to migrate away from using `svgs` and use the svg directly so instead of importing `svgs` here, import:

```
'common/views/svg',
 'inlineSvg!svgs/icon/hand.svg',
 'inlineSvg!svgs/icon/arrowRight.svg',
```

and replace ` svgs('arrowRight')` and ` svgs('hand')` with `svg(arrowRight)` and `svg(hand)`.